### PR TITLE
Merge MI-26 with dev 

### PR DESCRIPTION
**Pull request by AlineTheAlien**
Currently a bug present with SetGSD (#65), will fix that on a new branch, with another pull request

### CODE REVIEW

File: mRides-app/Resources/layout/UserProfile.axml [hunk 1]
```diff
@@ -35,11 +35,35 @@
...
-            <ImageView
-                android:src="@android:drawable/ic_menu_gallery"
-                android:layout_width="55.0dp"
-                android:layout_height="45.0dp"
-                android:id="@+id/genderImage" />
+            <LinearLayout
+                android:orientation="vertical"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:id="@+id/linearLayout1"
```
Comment (NassimElSayed) [inline mRides-app/Resources/layout/UserProfile.axml]: @AlineTheAlien I see some ids that are too generic, it might create conflicts in the future

Comment (AlineTheAlien) [inline mRides-app/Resources/layout/UserProfile.axml]: I completely missed that, thank you!

File: UnitTests/MappersTest/UserMapperTests.cs [hunk 2]
```diff
@@ -181,5 +182,65 @@ public void LeaveReview()
             }
             Assert.True(successRiderToDriverFeedback);
         }
+
+        /**
+        * Test whether we are able to successfully get the GSD of a user
+        */
+        [Test]
+        public void GetGSDTest()
```
Comment (nim442) [inline UnitTests/MappersTest/UserMapperTests.cs]: It might be better to name your tests the way they are named on the server for consistency's sake.

Comment (AlineTheAlien) [inline UnitTests/MappersTest/UserMapperTests.cs]: Yes you're right I didn't think of that

**Pull request merged**